docs/related-tools/codacy-plugin-tools.md

```diff
@@ -140,7 +140,11 @@ The Codacy GitHub repositories list the version and extra plugins supported by e
 <td><a href="https://github.com/codacy/codacy-pylint" class="skip-vale">codacy/codacy-pylint</a></td>
 </tr>
 <tr>
-<td><a href="https://github.com/remarkjs/remark-lint">RemarkLint</a></td>
+<td><a href="https://github.com/markdownlint/markdownlint">RemarkLint</a></td>
+<td><a href="https://github.com/codacy/codacy-markdownlint" class="skip-vale">codacy/codacy-markdownlint</a></td>
+</tr>
+<tr>
+<td><a href="https://github.com/remarkjs/remark-lint">MarkdownLint</a></td>
```

Comment:
The name of the tools and our corresponding repositories are mixed up:

![image](https://user-images.githubusercontent.com/60105800/119946142-375c7700-bf8e-11eb-91aa-e3fb2414e0c4.png)

```suggestion
<td><a href="https://github.com/markdownlint/markdownlint">markdownlint</a></td>
<td><a href="https://github.com/codacy/codacy-markdownlint" class="skip-vale">codacy/codacy-markdownlint</a></td>
</tr>
<tr>
<td><a href="https://github.com/remarkjs/remark-lint">RemarkLint</a></td>
```